experimental/packages/opentelemetry-instrumentation/test/common/third-party/node-semver/range-include.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Imported from https://github.com/npm/node-semver/blob/868d4bbe3d318c52544f38d5f9977a1103e924c2/test/fixtures/range-include.js
+
```

Comment:
Actually I think this block should be removed. My read of the intent of the "stored unmodified in a designated third-party folder" is that they should be completely unchanged from the source repo, if possible. Then we don't need to apply the OTel copyright and license to these files.

Reply:
Ah, ok. it is just left over. Just removed.